recipes/dscribe/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "dscribe" %}
+{% set version = "0.2.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7f232e6572a9815d232003e85ba714235cd203b9b3a7749b7133c28acf325b3e
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<37]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - scipy
+    - ase
+    - future
+    - scikit-learn
+    - joblib
+
+test:
+  imports:
+    - dscribe
+
+about:
+  home: https://singroup.github.io/dscribe/
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
```

From the manifest it looks like it might be included in the tarball?
```suggestion
  license_file: LICENSE.txt
```

---

No it is not included - I opened a pull request to include it in the future. https://github.com/SINGROUP/dscribe/pull/26

---

Ah, there is a typo in the manifest! 👍